frontend/packages/console-app/src/components/oauth-config/IdentityProviders.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { EmptyBox } from '@console/internal/components/utils';
+import { IdentityProvider } from '@console/internal/module/k8s';
+
+export const IdentityProviders: React.FC<IdentityProvidersProps> = ({ identityProviders }) => {
+  const { t } = useTranslation();
+  return _.isEmpty(identityProviders) ? (
+    <EmptyBox label={t('public~Identity providers')} />
```

Comment:
i8n namespaces need to be updated in moved files:
```suggestion
    <EmptyBox label={t('console-app~Identity providers')} />
```